### ISSUE TITLE
'gen_nn_ops' is not defined error - using 'selu' activation

### ISSUE THREAD

**Issue by RoboKiteCat** [apple/tensorflow_macos#103]
When fitting a simple neural network using the 'selu' activation an error is produced, as follows:

~/opt/anaconda3/envs/python38AppleTensor/lib/python3.8/site-packages/tensorflow/python/keras/activations.py in selu(x)
    194 #--- APPLE_MLCOMPUTE ---
    195   if mlcompute.enable_mlc_op(input_tensors=[x]):
--> 196     return gen_nn_ops.selu(x)
    197 #--- APPLE_MLCOMPUTE ---
    198   return nn.selu(x)

'gen_nn_ops' is not defined 

Fitting the same nn but instead of 'selu' use any of 'relu' or 'tanh' or 'sigmoid' does not produce the error.

Configuration:

from tensorflow.python.compiler.mlcompute import mlcompute
mlcompute.set_mlc_device(device_name='cpu')
import tensorflow.compat.v2 as tf
tf.enable_v2_behavior()

Tested on iMac 2020 i9 64Gb with AMD Radeon Pro 5700 XT 16 GB

**Comment by anna-tikhonova** [comment]
Thank you very much for reporting this issue. Could you please provide a reproducible test case, so we can investigate this issue locally?

**Comment by RoboKiteCat** [comment]
Hello,

Sorry its taken me a long time to come back to you.  I have now tested my original code thoroughly, now against **alpha2**, running on both intel i9 (cpu only) and M1.  In these configurations I was **not** able to reproduce the issue; selu activation was found to work.  Since the original issue occurred with the 'cpu' flag enabled I have not tested against GPU (though very similar code worked fine).

So I conclude that either the issue is fixed in alpha 2, or more likely my original environment was not well configured and in establishing the apha2 environment it is better / more complete.  I believe I was also previously importing keras.layers instead of tensorflow.keras.layers.

For interest the i9 takes around 3 minutes 16 seconds (maxing CPU) whereas the M1 takes 2 minutes 11 seconds.  Very similar code on the i9 but using 5700XT 16Gb GPU (and relu activation) takes around 17 minutes.

Thank you for all for your work on this fork.

**Comment by anna-tikhonova** [comment]
Thank you!
